### PR TITLE
Add missing spanish translations from new items, Dofus Effects, Prysm

### PR DESCRIPTION
**Pull request by rogerpb98**
…as translations + Effects

### CODE REVIEW

File: server/app/database/data/items.json [hunk 1]
```diff
@@ -124872,10 +124975,10 @@
   {
     "dofusID": 10345,
     "name": {
-      "en": "Turpiude Band",
+      "en": "Turpitude Band",
```
Comment (lightyearsaway) [inline server/app/database/data/items.json:124978]: lol

Comment (lightyearsaway) [reply server/app/database/data/items.json:124978]: Leave this as it was for now, we'll fix it in another pull request.

File: server/app/database/data/spells.json [hunk 2]
```diff
@@ -24509,7 +24509,7 @@
                       "en": "1 cell from center of trap",
                       "fr": "1 case d'éloignement",
                       "de": "1 cell from center of trap",
-                      "es": "1 cell from center of trap",
+                      "es": "a 1 casilla del centro de la trampa",
```
Comment (lightyearsaway) [inline server/app/database/data/spells.json:24512]: This is used here Fragmentation Trap - you think the first letter should stay lowercase?

File: server/app/database/data/spells.json [hunk 3]
```diff
@@ -84051,7 +84051,7 @@
                       "en": "4 allies",
                       "fr": "4 alliés",
                       "de": "4 allies",
-                      "es": "4 allies",
+                      "es": "4  aliados",
```
Comment (lightyearsaway) [inline server/app/database/data/spells.json:84054]: Looks like there's an extra space here.

**Comment by lightyearsaway** [comment]
I think it would be better if you could keep the order of the languages the same, so "es" before "pt" for all the Prysmas and the Dofuses and things like that. We should have filled out the placeholders for the languages we didn't have yet so that that would be more clear. I get it's kind of a lot of braindead busy work so it's OK if you don't want to do it, just keep in mind in the future.

**Comment by rogerpb98** [comment]
I noticed i cloned the master branch on visual studio and made this pull request to master, any way of changing that?

**Comment by lightyearsaway** [comment]
Awesome stuff. Thanks for the help!

**Pull request merged**